Formula/fnm.rb

```diff
@@ -0,0 +1,53 @@
+class Fnm < Formula
+  desc "Fast and simple Node.js version manager"
+  homepage "https://fnm.vercel.app"
+  url "https://github.com/Schniz/fnm/archive/v1.22.2.tar.gz"
+  sha256 "49013376359a1e0d690359b10c09604df90ab30602e8400acf0a97d241707344"
+  license "GPL-3.0-only"
+  head "https://github.com/Schniz/fnm.git"
+
+  livecheck do
+    url "https://github.com/Schniz/fnm/releases/latest"
+    regex(%r{href=.*?/tag/v?(\d+(?:\.\d+)+)["' >]}i)
+  end
+
+  depends_on "rust" => :build
+
+  uses_from_macos "zlib"
+
+  def install
+    system "cargo", "install", *std_cargo_args
+
+    (bash_completion/"fnm").write `#{bin}/fnm completions --shell bash`
+    (fish_completion/"fnm.fish").write `#{bin}/fnm completions --shell fish`
+    (zsh_completion/"_fnm").write `#{bin}/fnm completions --shell zsh`
```

Comment:
```suggestion
    (bash_completion/"fnm").write Utils.safe_popen_read("#{bin}/fnm completions --shell bash")
    (fish_completion/"fnm.fish").write Utils.safe_popen_read("#{bin}/fnm completions --shell fish")
    (zsh_completion/"_fnm").write Utils.safe_popen_read("#{bin}/fnm completions --shell zsh")
```

Reply:
👍 
probably need to change gh too (I copied from there): https://github.com/Homebrew/homebrew-core/blob/master/Formula/gh.rb#L28-L30